pkg/job/discovery.go

```diff
@@ -252,6 +263,8 @@ func getFilteredMetricDatas(
 					Period:                 m.Period,
 				})
 			}
+		} else {
+			logger.Warn("skipping resource unmatched by associator", "metric", m.Name, "dimensions", cwMetric.Dimensions)
```

Comment:
This is logged when a metric from `ListMetrics` does not match any resource. No resource is being skipped here, so "skipping resource unmatched by associator" is incorrect. Could this be something like "skipping metric with no matching resource" or similar?

In my setup, this scenario is also relatively common - it occurs any time I am only selecting a subset of resources in a discovery job with `searchTags`, in which case this message gets logged for every metric that relates to an unmatched resource (almost 700 times every scrape, with my current configuration). Could this log message be downgraded to debug, since it's somewhat normal for metrics to exist in cloudwatch that don't correspond to an AWS resource I'm targeting with tags?

Reply:
Done!